tsconfig.json

```diff
@@ -65,7 +65,7 @@
         // "newLine": "crlf",                                /* Set the newline character for emitting files. */
         // "stripInternal": true,                            /* Disable emitting declarations that have `@internal` in their JSDoc comments. */
         // "noEmitHelpers": true,                            /* Disable generating custom helper functions like `__extends` in compiled output. */
-        "noEmitOnError": true /* Disable emitting files if any type checking errors are reported. */,
+        "noEmitOnError": false /* Disable emitting files if any type checking errors are reported. */,
```

Comment:
Why switch to false?

Reply:
I have fixed this in a seperate PR - the problem was this feature meant the hot-reload local dev server would exit if there was a typescript error requiring a restart; I have fixed this and sent a PR with a fix for that.